debugger-guide.md

```diff
@@ -0,0 +1,62 @@
+
+# Debugger Guide 👨‍🔧
+
+Here we will learn how to use debugger for Keploy in VS Code.
+
+To bring up the **Run and Debug** view, select the **Run and Debug** icon in the **Activity Bar** on the side of VS Code. You can also use the keyboard shortcut `Cmd + Shift + D` for MacOS and `Ctrl + shift + D` for Windows.
+
+If running and debugging is not yet configured (no `launch.json` has been created), VS Code shows the Run start view.
+
+Click on **create a launch.json file**  👉  **Go**  👉  **Go: Launch Package**
+
+Navigate to `launch.json` to begin crafting JSON objects.
+
+```json
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Record",
+            "type": "go",
+            "request": "launch",
+            "mode": "auto",
+            "asRoot": true,
+            "console": "integratedTerminal",
+            "program": "main.go",
+            "args": [
+                "record",
+                "-c",
+                "<path_to_executable>"
+            ],
+        },
+        {
+            "name": "Test",
+            "type": "go",
+            "request": "launch",
+            "mode": "auto",
+            "asRoot": true,
+            "console": "integratedTerminal",
+            "program": "main.go",
+            "args": [
+                "test",
+                "-c",
+                "<path_to_executable>",
```

Comment:
I think we can also use the docker run command here and not just the path to the executable file